src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,25 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
+    private Comparator<Employee> employeeComparator = new Comparator<Employee>() {
```

Comment:
let's better make it a local variable
```suggestion
    private Comparator<Employee> employeeComparator = new Comparator<>() {
```